lib/rules/jsx-no-target-blank.js

```diff
@@ -15,7 +15,7 @@ module.exports = function(context) {
         var relFound = false;
         var attrs = node.parent.attributes;
         for (var idx in attrs) {
```

Comment:
Do you want this as part of this PR, or should it be a separate one?  Feels unrelated to the bug IMO, but I'm willing to do it if you want.